exercises/etl/canonical-data.json

```diff
@@ -38,7 +38,7 @@
         }
       },
       {
-        "description": "transforms the full set of scrabble tiles (multiple scores with differing numbers of letters)",
+        "description": "the full set of scrabble tiles (multiple scores with differing numbers of letters)",
```

Comment:
That they are scrabble tiles is irrelevant and leads to overly verbose test names since many tracks construct the test name from the description.
"multiple scores with differing numbers of letters" should be sufficient to describe the test.

If any additional description about the problem is required it should be in the `description.md` file